src/components/ListItem.jsx

```diff
@@ -1,5 +1,7 @@
 import { useState, useEffect } from 'react';
 import './ListItem.css';
+import { FaTrashCan } from 'react-icons/fa6';
+import { FaCartShopping } from 'react-icons/fa6';
```

Comment:
You can move both icon imports into one line since they are imported from the same collection